### PR TITLE
add CONTRIBUTING.md with CLA instructions

### PR DESCRIPTION
**Pull request by lwwmanning**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+2. Implement the change.
+    * If the change is large, consider posting a draft pull request (PR)
+      with the title prefixed with [WIP], and share with the team to get early feedback.
+    * Give the PR a clear, brief description; this will be the commit message
+      when the PR is merged.
+    * Make sure the PR passes all CI tests.
+
+3. Open a PR to indicate that the change is ready for review.
+    * Sign the [Contributor License Agreement](https://gist.github.com/lwwmanning/e246ddd906c18ffb79bfef61a888b080) (CLA) when prompted.
```
Comment (robert3005) [inline CONTRIBUTING.md]: is this correct?

Comment (gatesn) [reply CONTRIBUTING.md]: At least move it to the spiraldb org

Comment (lwwmanning) [reply CONTRIBUTING.md]: Orgs don't support gists, and cla-assistant requires a gist. 

https://github.com/cla-assistant/cla-assistant/issues/926

Comment (lwwmanning) [reply CONTRIBUTING.md]: I'll just change it to https://cla.spiraldb.com and we can manage via Cloudflare

**Comment by gatesn** [comment]
Just waiting on that link to work @lwwmanning

**Review by robert3005 — APPROVED** [review]

**Pull request merged**